feature-libs/pickup-in-store/assets/translations/en/pickupInStore.json

```diff
@@ -56,7 +56,7 @@
     "pickUpInStoreAddress": "Pick Up Store Address",
     "itemsToBeShipped": "Items To Be Shipped",
     "shippingAddress": "Shipping Address",
-    "deliveryMethod": "Delivery Method",
+    "deliveryMethod": "Delivery Options",
```

Comment:
Changing or removing translation keys or values is breaking change
to avoid breaking change add new translation key